Lib/venv/scripts/nt/activate.bat

```diff
@@ -1,7 +1,7 @@
 @echo off
 
 rem This file is UTF-8 encoded, so we need to update the current code page while executing it
-for /f "tokens=2 delims=:" %%a in ('"%SystemRoot%\System32\chcp.com"') do (
+for /f "tokens=2 delims=:." %%a in ('"%SystemRoot%\System32\chcp.com"') do (
```

Comment:
I suggested this on PR 10295, but there were concerns that it wasn't a robust solution.